server/poll/transform.go

```diff
@@ -127,6 +136,44 @@ func (p *Poll) makeAdditionalText(localizer *i18n.Localizer, numberOfVotes int)
 	return strings.Join(lines, "\n")
 }
 
+func progressBarStr(progress float64, width float64) string {
+	progress = math.Min(1.0, math.Max(0, progress))
+	wholeWidth := math.Floor(progress * width)
+	line := ""
+	for i := 0.0; i < wholeWidth; i++ {
+		line += "█"
+	}
+	for i := 0.0; i < (width - wholeWidth); i++ {
+		line += "░"
+	}
+
+	return line
+}
+
+func generateProgressBars(votes map[string]int, numberOfVotes int) []string {
+	lines := make([]string, 0)
+
+	keys := make([]string, 0)
+	for n := range votes {
+		keys = append(keys, n)
+	}
+
+	sort.Strings(keys)
+
+	for _, n := range keys {
+		v := votes[n]
+		var progress float64
+		if numberOfVotes > 0 {
+			progress = float64(v) / float64(numberOfVotes)
+		} else {
+			progress = 0
+		}
+
+		lines = append(lines, fmt.Sprintf("%s\t%s\t`%3d %%`", progressBarStr(progress, 32), n, int(progress*100.0)))
```

Comment:
```suggestion
		lines = append(lines, fmt.Sprintf("`%s`\t%s\t`%3d %%`", progressBarStr(progress, 32), n, int(progress*100.0)))
```

In my browser, `█` and `░` seem to have different width. My suggestion would fix this difference, and additionally could fix issues on mobile. 
![スクリーンショット 2020-01-07 22 26 47](https://user-images.githubusercontent.com/1453749/71900091-6e1acd00-31a0-11ea-82b3-aa5f6abe5142.png)

Reply:
What does the ``` do exactly? Good point about the width of these characters, it worked for the clients I tested it with. There are a few different symbols like this but, I think this is a font issue that will make it difficult to select characters that work the same in all fonts. Any ideas?

Reply:
I'm sorry I don't have any better ideas...

Surrounding words with `` `TEXT` `` is the format of Markdown, and is converted to `<span class="codespan__pre-wrap"><code>TEXT</code></span>` by Mattermost. In my local, although normal text is rendered with Open Sans font, `<code>` elements is rendered with monospace fonts (e.g. Menlo, Monaco, monospace, or so on). But as you pointed out, which font is actually used would depend on the user's environment. So my idea is just a workaround, but I think it would be some effective with less efforts.